rotkehlchen/globaldb/schema.py

```diff
@@ -108,8 +108,11 @@
 INSERT OR IGNORE INTO token_kinds(token_kind, seq) VALUES ('C', 3);
 """
 
-# Using asset_id as a primary key here since nothing else is guaranteed to be unique
-# Advantage we have here is that by deleting the parent asset this also gets deleted
+# The common_assets_details contains information common for all the crypto-assets
+# leaving to the assets table information that is shared among the custom assets.
+# For every assets row we have a row in the common_assets_details table or in the
+# custom_assets table. The identifier of the asset is primary key since this allows as to do
```

Comment:
Does primary key have to do anything with foreign keys?

Or do you mean primary key in the assets table? If yes, I would it closer to the assets table

Reply:
Also is there a need to mention foreign keys here? on delete cascade is not the primary property of the foreign keys, right? So I would either don't mention them or add other features that they offer. What do you think?

(same question about `evm_tokens` and `custom_assets`)

Reply:
Is just to make it clear that is used to relate with the other tables and how they are connected. No special but they are referenced by the asset identifer